### PR TITLE
Add header to rickroll post

### PR DESCRIPTION
**Pull request by rsaunders100**
Was not sure about the `excerpt` part.

Should it actually be a excerpt or more like a blurb

### CODE REVIEW

File: _posts/2017-05-12-how-to-un-rickroll-yourself.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---
+layout: post
+title:  "How to un-rickroll yourself"
+author: "Robert Saunders"
+excerpt: >
+  A story of how an entire morning can be lost if you forget to lock your laptop.
```
Comment (rsaunders100) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: Maybe

> A story of how an entire morning was lost when I left my laptop unlocked.

or

> A story of how leaving a laptop unlocked can lose the entire morning.
idk

Comment (rsaunders100) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: Or 

> Leaving your laptop unlocked can lose an entire morning.

Comment (rsaunders100) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: or

> How I disinfected Rick Astley from my laptop

**Comment by rsaunders100** [comment]
Also, should I update the published at date?

**Comment by rsaunders100** [comment]
Also, should I update the published at date?

File: _posts/2017-05-12-how-to-un-rickroll-yourself.md (hunk 1 above)
Comment (jphastings) [inline _posts/2017-05-12-how-to-un-rickroll-yourself.md]: How about both:

> A story of how leaving a laptop unlocked can lose the entire morning. Or: How I really, really wanted to give Rick Astley Up.

File: _posts/2017-05-16-how-to-un-rickroll-yourself.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+---
+layout: post
+title:  "How to un-rickroll yourself"
+author: "Robert Saunders"
+excerpt: >
+  A story of how leaving your laptop unlocked can lose the entire morning
+---
+
 # How to un-rickroll yourself
```
Comment (krissy) [inline _posts/2017-05-16-how-to-un-rickroll-yourself.md]: The `excerpt` value is what will appear on list pages and anywhere a snippet of your post shows up that isn't the actual post's page. It is also defaulted to being the first chunk of text on your page, i.e. so it presumes your excerpt is the first few lines of your blog post.

If you _don't_ want this to happen, e.g. your post summary is not the same as the first few lines of your post, you can also add a `hide_excerpt_on_post: true`.

You also don't need the `# How to un-rickroll yourself` on L9 anymore, as the title will be set automagically from the frontmatter (and will duplicate if you leave this in)

**Comment by krissy** [comment]
The image URL at the bottom is incorrectly referencing the folder of a past blog post: https://github.com/deliveroo/deliveroo.engineering/blob/87a0fba3eedca0b263def56aa7b6625018b2d748/images/posts/hackday-and-the-17pound-soda/rick-roll.jpg

This needs updating too.

Other than these, happy I can share this with all the non-Roos now! :D

**Comment by rsaunders100** [comment]
@krissy - Thank you! :)

**Pull request merged**